### DISCUSSION TITLE
9BD18B53-18B6-4CDE-B632-B5C6C38C8AA5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#95] [General]
# Methane Molecule

his adds two ball and stick models of a Methane and Caffeine molecule in orbit around Earth. One is at the actual size of a Methane Molecule (being 0.38 nm) and one magnified 100000000x to accommodate for Celestia being unable to zoom in far enough to view an object in the nanometer range. Model sourced from World of Molecule's Jsmol 3D molecule viewer and converted from X3D to CMOD by Anthony_B_Russo10.

https://celestia.mobi/addon?item=9BD18B53-18B6-4CDE-B632-B5C6C38C8AA5

**Comment by Anonymous-being** [comment]
why? just why
